node/malus/integrationtests/0001-dispute-valid-block.feature

```diff
@@ -16,14 +16,14 @@ bob: reports block height is at least 2
 bob: reports peers count is at least 2
 charlie: reports block height is at least 2
 charlie: reports peers count is at least 2
-alice: reports parachain_candidate_disputes_total is at least 1 within 250 seconds
-bob: reports parachain_candidate_disputes_total is at least 1 within 90 seconds
-charlie: reports parachain_candidate_disputes_total is at least 1 within 90 seconds
-alice: reports parachain_candidate_dispute_votes{validity="valid"} is at least 1 within 90 seconds
-bob: reports parachain_candidate_dispute_votes{validity="valid"} is at least 2 within 90 seconds
-charlie: reports parachain_candidate_dispute_votes{validity="valid"} is at least 2 within 90 seconds
-alice: reports parachain_candidate_dispute_concluded{validity="valid"} is at least 1 within 90 seconds
-alice: reports parachain_candidate_dispute_concluded{validity="invalid"} is 0 within 90 seconds
-bob: reports parachain_candidate_dispute_concluded{validity="valid"} is at least 1 within 90 seconds
-charlie: reports parachain_candidate_dispute_concluded{validity="valid"} is at least 1 within 90 seconds
-charlie: reports parachain_candidate_dispute_concluded{validity="valid"} is at least 1 within 90 seconds
+alice: reports polkadot_parachain_candidate_disputes_total is at least 1 within 250 seconds
```

Comment:
this doesn't need to be updated I believe cc @pepoviola

Reply:
It needs. It failed  before.

Reply:
Let's wait what the test is saying.

Reply:
Hi @ordian, no. The prefix isn't expected.
Thx!

Reply:
@pepoviola it seems there are some hard coded metrics. See the tests below. Can you fix this?

Reply:
Hi @bkchr, yes. I will fix asap.
Thanks!

Reply:
Can you report back here? Then I will wait with merging this pr

Reply:
@bkchr, fixed in https://github.com/paritytech/polkadot/pull/3623/commits/877a20bbe3a60d77e35afc4006e6eebb74f9b344.
Thanks!

Reply:
Test are failing, let me review how the metrics are reported with this changes.
Thx!

Reply:
Hi @bkchr, I check why the test are failing and with this changes metrics are reported like:
```
substrate_sub_libp2p_peers_count{chain="/cfg/wococo-local.json"} 5
```

The issue is the suffix  `{chain="/cfg/wococo-local.json"}` is this needed? Currently the metrics are reported without that.

```
polkadot_sub_libp2p_peers_count 3
```

Zombiente will try to get the metric without looking the prefix (polkadot/substrate) but don't have the logic to add the chain suffix.

Thanks!

Reply:
> The issue is the suffix {chain="/cfg/wococo-local.json"}

This is a  bug and should be only `chain=wococo`. I'm working on a fix. However, this `chain` arg stays.

Reply:
Oks, this will need some changes in `zombienet` to match the metrics. I will work on that asap and update the runner image in `CI` to review.
Thanks!

Reply:
Hi @bkchr, I already added support for the suffix in `zombienet` and now test are ok.
Thanks!